lib/github/api.nit

```diff
@@ -1122,6 +1131,10 @@ class GithubDeserializer
 			return "Branch"
 		else if raw.has_key("total_count") and raw.has_key("items") then
 			return "SearchResults"
+		else if raw.has_key("total") and raw.has_key("weeks") then
+			return "ContributorStats"
+		else if raw.has_key("a") and raw.has_key("d") and raw.has_key("c") then
```

Comment:
I could have named those fields!

Reply:
Actually, there is a striking resemblance between you and Tenderlove... At least for the bad jokes: p

https://twitter.com/tenderlove/status/714540289675763712?lang=en